Python/cim-convert-tool/configEqCim17.py

```diff
@@ -0,0 +1,602 @@
+                'cim:ConductingEquipment.BaseVoltage': {'rdf:resource': {'type': 'string'}, 'list': False}
+            }
+        },
+        'cim:LoadResponseCharacteristic': {
+            'mainAttributes': {
+                'rdf:ID': {'type': 'string', 'list': False} # @id
+            },
+            'tags': {
+                'cim:IdentifiedObject.mRID': {'type': 'string', 'list': False},
+                # 'cim:IdentifiedObject.aliasName': {'type': 'string', 'list': False},
+                'cim:IdentifiedObject.description': {'type': 'string', 'list': False},
+                'cim:IdentifiedObject.name': {'type': 'string', 'list': False},
+                'cim:LoadResponseCharacteristic.pConstantPower': {'type': 'float', 'list': False},
+                'cim:LoadResponseCharacteristic.pConstantCurrent': {'type': 'float', 'list': False},
+                'cim:LoadResponseCharacteristic.pConstantImpedance': {'type': 'float', 'list': False},
+                'cim:LoadResponseCharacteristic.qConstantPower': {'type': 'float', 'list': False},
+                'cim:LoadResponseCharacteristic.qConstantCurrent': {'type': 'float', 'list': False},
+                'cim:LoadResponseCharacteristic.qConstantImpedance': {'type': 'float', 'list': False},
+                'cim:LoadResponseCharacteristic.exponentModel': {'type': 'bool', 'list': False}
+            },
+            'attributes': {
+
+            }
+        },
+        'cim:OperationalLimitSet': {
+            'mainAttributes': {
+                'rdf:ID': {'type': 'string', 'list': False} # @id
+            },
+            'tags': {
+                'cim:IdentifiedObject.mRID': {'type': 'string', 'list': False},
+                # 'cim:IdentifiedObject.aliasName': {'type': 'string', 'list': False},
+                'cim:IdentifiedObject.description': {'type': 'string', 'list': False},
+                'cim:IdentifiedObject.name': {'type': 'string', 'list': False}
+            },
+            'attributes': {
+                'cim:OperationalLimitSet.Terminal': {'rdf:resource': {'type': 'string'}, 'list': False}
+            }
+        },
+        'cim:OperationalLimitSet': {
+            'mainAttributes': {
+                'rdf:ID': {'type': 'string', 'list': False} # @id
+            },
+            'tags': {
+                'cim:IdentifiedObject.mRID': {'type': 'string', 'list': False},
+                # 'cim:IdentifiedObject.aliasName': {'type': 'string', 'list': False},
+                'cim:IdentifiedObject.description': {'type': 'string', 'list': False},
+                'cim:IdentifiedObject.name': {'type': 'string', 'list': False},
+                'cim:OperationalLimitType.acceptableDuration': {'type': 'float', 'list': False, 'CIMDatatype': 'cim:Seconds.value'},
+                'cim:OperationalLimitType.isInfiniteDuration': {'type': 'bool', 'list': False}
+            },
+            'attributes': {
```

Comment:
Should include reference to OperationalLimitType.kind and OperationalLimitType.direction.  Reference to terminal is not used.